platform/switch/os_switch.cpp

```diff
@@ -478,15 +477,15 @@ bool OS_Switch::has_virtual_keyboard() const {
 }
 
 int OS_Switch::get_virtual_keyboard_height() const {
-	// todo: actually figure this out
 	if (!g_swkbd_open) {
 		return 0;
 	}
-	return 300;
+	return 400;
 }
 
-void OS_Switch::show_virtual_keyboard(const String &p_existing_text, const Rect2 &p_screen_rect, int p_max_input_length) {
-	if (!g_swkbd_open) {
+void OS_Switch::show_virtual_keyboard(const String &p_existing_text, const Rect2 &p_screen_rect, bool p_multiline, int p_max_input_length, int p_cursor_start, int p_cursor_end) {
+	printf("Showing kbd!\n");
```

Comment:
This doesn't need a debug print, now that it's fixed.

Reply:
Agreed, will fix.